README.md

```diff
@@ -1,68 +1,51 @@
-# EVE
 
-This project aims to simulate basic lifeforms evolving and getting smarter, in a simplified manner.
+# EVE
 
-## Development
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
```

Comment:
Why experimental & unstable?

Reply:
To mark it as non-working - it shouldn't look like this is a fully developed product, or works in the first place (apart from being compilable)

Reply:
yeah, but then choose one